docker-compose.yml

```diff
@@ -191,6 +191,17 @@ services:
       - FLINKSTER_CLIENT_ID=${FLINKSTER_CLIENT_ID:?missing/empty}
       - FLINKSTER_SECRET=${FLINKSTER_SECRET:?missing/empty}
     restart: unless-stopped
+    healthcheck:
+      test: ps aux | grep -q 'x2gbfs\.x2gbfs' || exit 1
```

Comment:
This health check checks neither
- if x2gbfs is actually healthy (just if a process exists); nor
- if all feeds have been generated (we would use a special file, e.g. `/var/run/x2gbfs-feeds-generated-once` to signal this).

I highly recommend against having an unmeaningful health check because it gives a false sense of stability!